ladybug_comfort/collection/adaptive.py

```diff
@@ -0,0 +1,431 @@
+# coding=utf-8
+"""Object for calculating Adaptive comfort from DataCollections."""
+from __future__ import division
+
+from ..adaptive import adaptive_comfort_ashrae55, adaptive_comfort_en15251, \
+    get_adaptive_comfort_conditioned_function, cooling_effect_ashrae55, \
+    cooling_effect_en15251, get_operative_temperature, \
+    weighted_running_mean_hourly, weighted_running_mean_daily
+from ..parameter.adaptive import AdaptiveParameter
+from ._base import ComfortCollection
+
+from ladybug._datacollectionbase import BaseCollection
+from ladybug.datacollection import HourlyContinuousCollection, DailyCollection, \
+    MonthlyCollection, MonthlyPerHourCollection, HourlyDiscontinuousCollection
+from ladybug.analysisperiod import AnalysisPeriod
+
+from ladybug.datatype.temperature import Temperature, OperativeTemperature, \
+    PrevailingOutdoorTemperature
+from ladybug.datatype.speed import Speed, AirSpeed
+from ladybug.datatype.thermalcondition import ThermalComfort, ThermalCondition
+from ladybug.datatype.temperaturedelta import OperativeTemperatureDelta
+
+
+class Adaptive(ComfortCollection):
+    """Adaptive comfort DataCollection object.
+
+    Properties:
+        prevailing_outdoor_temperature
+        operative_temperature
+        air_speed
+        comfort_parameter
+        neutral_temperature
+        degrees_from_neutral
+        is_comfortable
+        thermal_condition
+        cooling_effect
+        percent_comfortable
+        percent_uncomfortable
+        percent_neutral
+        percent_hot
+        percent_cold
+    """
+    _model = 'Adaptive'
```

Comment:
I realize this seems duplicated for the Adaptive model but, in all other comfort models, this is different from the class name.  For example, the class is called `PMV `but the `_model `is called `'Predicted Mean Vote'`.   In other words, this attribute is here so that we have the full name of the model stored somewhere since most of the objects use abbreviations of the model name.